uk.co.compucorp.civicrm.tasksassignments/js/test/filters/due_test.js

```diff
@@ -24,14 +28,57 @@ define([
         from: new Date(today.getTime() + oneDay),
         until: new Date(today.getTime() + oneDay * 2)
       };
+
+      emptyDateRange = {
+        from: null,
+        until: null
+      };
+
+      emptyFromDateAndUntilDateInTheFuture = {
+        from: null,
+        until: new Date(today.getTime() + oneDay * 2)
+      };
+
+      emptyFromDateAndUntilDateInPast = {
+        from: null,
+        until: new Date(today.getTime() - oneDay * 2)
+      };
+
+      emptyFromDateAndUntilDateIsToday = {
+        from: null,
+        until: today
+      };
+
+      emptyUntilDateAndFromDateInThePast = {
+        from: new Date(today.getTime() - oneDay * 2),
+        until: null
+      };
+
+      emptyUntilDateAndFromDateIsToday = {
+        from: today,
+        until: null
+      };
+
+      emptyUntilDateAndFromDateInTheFuture = {
+        from: new Date(today.getTime() + oneDay * 2),
+        until: null
+      };
     }));
 
     describe('due date is in the future', function() {
-      var mock;
+      var mock, mockPlusTwo, mockPlusThree;
```

Comment:
I would suggest `var mockDueDate, mockPlusTwoDays, mockPlusThreeDays;` to make it more readable.